{
	"id": "63849",
	"key": "TIMOB-3217",
	"fields": {
		"issuetype": {
			"id": "1",
			"description": "A problem which impairs or prevents the functions of the product.",
			"name": "Bug",
			"subtask": false
		},
		"project": {
			"id": "10153",
			"key": "TIMOB",
			"name": "Titanium SDK/CLI",
			"projectCategory": {
				"id": "10100",
				"description": "Titanium and related SDKs used in application development",
				"name": "Client"
			}
		},
		"fixVersions": [],
		"resolution": {
			"id": "2",
			"description": "The problem described is an issue which will never be fixed.",
			"name": "Won't Fix"
		},
		"resolutiondate": "2016-08-19T18:59:14.000+0000",
		"created": "2011-04-15T03:39:44.000+0000",
		"priority": {
			"name": "Low",
			"id": "4"
		},
		"labels": [],
		"versions": [],
		"issuelinks": [],
		"assignee": {
			"name": "ingo",
			"key": "ingo",
			"displayName": "Ingo Muschenetz",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"updated": "2017-03-16T21:31:05.000+0000",
		"status": {
			"description": "The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.",
			"name": "Closed",
			"id": "6",
			"statusCategory": {
				"id": 3,
				"key": "done",
				"colorName": "green",
				"name": "Done"
			}
		},
		"components": [
			{
				"id": "10206",
				"name": "iOS",
				"description": "iOS Platform"
			},
			{
				"id": "10207",
				"name": "Tooling"
			}
		],
		"description": "Hi guys,\r\n\r\nI'm experiencing this error for every project that I try to compile in SDK 1.6:\r\n\r\n{code}\r\n[INFO] One moment, building ...\r\n[INFO] Titanium SDK version: 1.6.0\r\n[INFO] iPhone Device family: iphone\r\n[INFO] iPhone SDK version: 4.1\r\n[INFO] iPhone simulated device: iphone\r\n[ERROR]\r\n[ERROR] Error: Traceback (most recent call last): File \"/Library/Application Support/Titanium/mobilesdk/osx/1.6.0/iphone/builder.py\", line 1003, in main\r\nexecute_xcode(\"iphonesimulator%s\" % link_version,[\"GCC_PREPROCESSOR_DEFINITIONS=LOGID__=%s DEPLOYTYPE=development TI_DEVELOPMENT=1 DEBUG=1 TI_VERSION=%s\" % (log_id,sdk_version)],False)\r\nFile \"/Library/Application Support/Titanium/mobilesdk/osx/1.6.0/iphone/builder.py\", line 924, in execute_xcode\r\noutput = run.run(args,False,False,o)\r\nFile \"/Library/Application Support/Titanium/mobilesdk/osx/1.6.0/iphone/run.py\", line 39, in run\r\nsys.exit(rc)\r\nSystemExit: 1\r\n{code}\r\n\r\nAny idea how to get around this problem?? I can't build anything anymore :(",
		"attachment": [],
		"flagged": false,
		"summary": "iPhone Titanium SDK 1.6 Error on ANY project",
		"creator": {
			"name": "petersweeney",
			"key": "petersweeney",
			"displayName": "Peter Sweeney",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"subtasks": [],
		"reporter": {
			"name": "petersweeney",
			"key": "petersweeney",
			"displayName": "Peter Sweeney",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"environment": null,
		"comment": {
			"comments": [
				{
					"id": "130317",
					"author": {
						"name": "ryan",
						"key": "ryan",
						"displayName": "ryan",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "{html}<div><p>I too am experiencing the same issue. I'm new to appcellerator\nand as one of my first projects i'm trying out the tibar barcode\ntutorial: <a href=\n\"http://code.google.com/p/tibar/\">http://code.google.com/p/tibar/</a><br></p>\n<p>Below is the error from the Build.log of my project. Any help\nwould be appreciated. Thanks</p>\n<p><strong>BUILD FAILED</strong></p>\n<p>The following build commands failed:<br>\nbarcode:<br></p>\n<pre>\n<code>Ld /Users/.../Desktop/barcode/build/iphone/build/Debug-iphonesimulator/barcode.app/barcode normal i386</code>\n</pre>\n<p>(1 failure)</p>\n<p>EXIT CODE WAS: 1</p>\n<p>Exception detected in script:<br>\nTraceback (most recent call last):<br>\nFile \"/Library/Application\nSupport/Titanium/mobilesdk/osx/1.6.0/iphone/builder.py\", line 1003,\nin main</p>\n<pre>\n<code>execute_xcode(\"iphonesimulator%s\" % link_version,[\"GCC_PREPROCESSOR_DEFINITIONS=__LOG__ID__=%s DEPLOYTYPE=development TI_DEVELOPMENT=1 DEBUG=1 TI_VERSION=%s\" % (log_id,sdk_version)],False)</code>\n</pre>\n<p>File \"/Library/Application\nSupport/Titanium/mobilesdk/osx/1.6.0/iphone/builder.py\", line 924,\nin execute_xcode</p>\n<pre>\n<code>output = run.run(args,False,False,o)</code>\n</pre>\n<p>File \"/Library/Application\nSupport/Titanium/mobilesdk/osx/1.6.0/iphone/run.py\", line 39, in\nrun</p>\n<pre>\n<code>sys.exit(rc)</code>\n</pre>\n<p>SystemExit: 1</p></div>{html}",
					"updateAuthor": {
						"name": "ryan",
						"key": "ryan",
						"displayName": "ryan",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2011-04-15T03:39:45.000+0000",
					"updated": "2011-04-15T03:39:45.000+0000"
				},
				{
					"id": "130318",
					"author": {
						"name": "petersweeney",
						"key": "petersweeney",
						"displayName": "Peter Sweeney",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "{html}<div><p>The solution is this.. any existing project can't seem to cope\nwith being 'upgraded' to the new one.. You need to create an\nentirely new Project and copy over the resources etc for it to work\nagain.</p>\n<p>That's the only work around I found to this problem.</p></div>{html}",
					"updateAuthor": {
						"name": "petersweeney",
						"key": "petersweeney",
						"displayName": "Peter Sweeney",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2011-04-15T03:39:45.000+0000",
					"updated": "2011-04-15T03:39:45.000+0000"
				},
				{
					"id": "393644",
					"author": {
						"name": "cbarber",
						"key": "cbarber",
						"displayName": "Chris Barber",
						"active": true,
						"timeZone": "America/Chicago"
					},
					"body": "The Python-based build scripts have been deprecated and replaced by a new build system release with Titanium SDK 3.0.0, which was released on 12/14/2012. Resolving as invalid.",
					"updateAuthor": {
						"name": "cbarber",
						"key": "cbarber",
						"displayName": "Chris Barber",
						"active": true,
						"timeZone": "America/Chicago"
					},
					"created": "2016-08-19T18:59:14.000+0000",
					"updated": "2016-08-19T18:59:14.000+0000"
				},
				{
					"id": "413104",
					"author": {
						"name": "lmorris",
						"key": "lmorris",
						"displayName": "Lee Morris",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"body": "Closing ticket as the issue won't fix.",
					"updateAuthor": {
						"name": "lmorris",
						"key": "lmorris",
						"displayName": "Lee Morris",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2017-03-16T21:31:05.000+0000",
					"updated": "2017-03-16T21:31:05.000+0000"
				}
			],
			"maxResults": 4,
			"total": 4,
			"startAt": 0
		}
	}
}